Raptacon/Robot-2023
# Anisa balance

## Types of changes

## Associated Issue(s)
Issue: #68 

## Steps to Test

1.  make sure its off balance
2.  click x on xbox controller
3. it should auto balance


## Changelog

Added balance class, and added code to configBased robot that instantiates balance class and replacement drivetrain for the balance object.

## What?

Makes the robot auto balance on the charge station.

## Why?
This helps driveteam balance the robot by pressing and holding the x on the controller and could be implememnted into the autonomous so it autobalances in autonomous mode.
